### ISSUE TITLE
Line numbers are inconsistent and are affected by whitespace and comments in the file

### ISSUE THREAD

**Issue by fnimick** [sbdchd/squawk#373]
This causes issues with integration into other tools (such as an in-editor linter) since it doesn't line up with the actual code.

Examples:
```sql
create table test_table (
    test_column varchar(255)
);
```

This is fine, squawk reports: `test.sql:1:0: warning: prefer-text-field...`

However, when I add a blank line before this:
```sql
<blank>
create table test_table (
    test_column varchar(255)
);
```

This is now reported as `test.sql:1:1: ...` - should be `2:0`, but is instead line 1, column 1.

If I add a comment to the blank line:
```sql
-- foo
create table test_table (
    test_column varchar(255)
);
```

Now it's reported as `test.sql:1:0` - line 1, column 0 again, where it should be `2:0`. Any amount of whitespace between the comment and the statement has no effect.

For the final example:
```sql
-- foo

select foobar;

create table test_table (
    test_column varchar(255)
);
```

This reports on `test.sql:4:2`. Adding whitespace between the comment and the select increments the reported error row, and adding whitespace between the select and the create table statement increments the reported error column. e.g.

```sql
-- foo


select foobar;



create table test_table (
    test_column varchar(255)
);
```
reports on `test.sql:5:4`

**Comment by sbdchd** [comment]
This should be fixed by squawk 2.0.0 which includes a new parser
